src/util/codeUtil.ts

```diff
@@ -94,6 +94,12 @@ export class CodeUtil {
         await this.checkCodeVersion(version);
 
         const literalVersion = version === 'latest' ? this.availableVersions[0] : version;
+        if(this.releaseType == ReleaseQuality.Stable && literalVersion !== this.availableVersions[0]) {
+            console.log(`
+                WARNING: You are using the obsolete VSCode version '${literalVersion}'. The latest stable version is '${this.availableVersions[0]}'.
```

Comment:
Isn't 'obsolete' a bit too much? How about 'outdated' or there are other implications?

Reply:
you are absolutely right. I dont know why I used obsolete. The outdated is much more precise. I will change it. Thank you!